data/korath/korath missions.txt

```diff
@@ -61,3 +61,353 @@ mission "Discovered Korath Space"
+			choice
+				`	"I understand, and the money isn't a huge problem. I was just curious. Load it up onto my ship, and I'll be ready to go."`
+					goto agree
+				`	"Oh. Well, I'll make an exception too. Load it up, and I'll head out."`
+					goto agree
+				`	"What exactly would I be transporting?"`
+					goto whatsinside
+				`	"Sorry, but I can't make that trip for only <payment>. You'll have to find someone else."`
+					goto nocando
+			label nocando
+			`	The Wanderer deflates a little when it hears your response. "Oh. I understand. Thank you for your time, Captain <last>," it says, then it wanders off to look for another pilot.`
+				decline
+			label agree
+			`	The Wanderer is happy to hear your response. "Wonderful! I will ensure the supplies are loaded onto your ship [immediately, quickly]. Thank you, Captain <last>!" It hurries off to a group of Wanderers nearby and starts barking orders. The group breaks up, and within minutes they're at your ship loading the supplies.`
+				accept
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 5000
+		conversation
+			`When you arrive on <planet>, a group of Korath is waiting to unload the cargo. You try and imitate their gesture of greeting, holding your hands up with your palms toward the group. The look surprised but hiss approvingly.`
+			`	Soon, all the supplies are off of your ship. The Korath thank you, then they begin unpacking it right in the middle of the spaceport. Already, a crowd has begun to gather around the equipment. It looks like it will be put to good use here. When you return to your ship, you see the Wanderer who offered you the job has transferred you your payment of <payment>.`
+				
+mission "Korath Family to Ringworld"				
+	name "Korath to <planet>"
+	description "A Korath family of <bunks> has paid you <payment> to take them to the Quarg ringworld of <destination>."
+	minor
+	passengers 3
+	source "Laki Nemparu"
+	destination "Kuwaru Efreti"
+	to offer
+		random < 60
+	on offer
+		conversation
+			`The smog is so thick on Laki Nemparu that even the brightest hours of the day feel like evening to you. Most of the Korath are a few shades darker than those of other planets, more a result of dirtiness rather than skin pigmentation. Within minutes, a small layer of soot clings to your skin as well.`
+			`	As you observe your surroundings, a Kor Efret approaches you followed by two others, one likely a child, based on its size, and the other hunched over a cane it uses for support - probably elderly. The one in front speaks to you rapidly in its own language before realizing you don't understand it at all. It pauses to take a deep breath, then erupts into a coughing fit as it inhales far too much smog. The older Korath behind begins coughing too, and it's almost a full minute before they've both recovered.`
```

Comment:
why would that Kor ever assume I'd understand its language? also, earlier you used `her` and `she` for the Korath that's working with Rek, so I take it the Korath aren't like the Quarg (in that the Quarg are referred to as `it`), so maybe use `they` for this Kor or give them a gender.